MSAL/src/native_auth/network/errors/MSALNativeAuthErrorMessage.swift

```diff
@@ -37,6 +37,10 @@ enum MSALNativeAuthErrorMessage {
     static let passwordRequiredNotImplemented = "Implementation of onSignInPasswordRequired required"
     static let codeRequiredNotImplemented = "Implementation of onSignInCodeRequired required"
     static let codeRequiredForPasswordUserLog = "This user does not have a password associated with their account. SDK will call `delegate.onSignInCodeRequired()` and the entered password will be ignored"
+
+    static func requiredDelegateMethod(_ method: String) -> String {
+        return "Implementation of \(method) required to continue this flow"
```

Comment:
Is there a specific reason you didn't use the "%@" as we did for `attributeValidationFailed` string? 
Given that we have this generic string/method can you please unify `passwordRequiredNotImplemented` and `codeRequiredNotImplemented`?
I'd also remove `to continue this flow` to this message

Reply:
Done. Sorry I had forgotten about the %@, I just remember that @rodhan-ms told me this in a previous PR!